lightning/src/onion_message/blinded_route.rs

```diff
@@ -113,6 +114,52 @@ fn encrypt_payload<P: Writeable>(payload: P, encrypted_tlvs_ss: [u8; 32]) -> Vec
 	writer.0
 }
 
+impl Writeable for BlindedRoute {
+	fn write<W: Writer>(&self, w: &mut W) -> Result<(), io::Error> {
+		self.introduction_node_id.write(w)?;
+		self.blinding_point.write(w)?;
+		for hop in &self.blinded_hops {
```

Comment:
Looks like we'll want to write the number of hops as a `u8` in order to mach the encoding used in the `offer` message. Not in the latest spec draft, but see discord discussion.

Reply:
Updated to this